src/xrt/drivers/illixr/illixr_device.cpp

```diff
@@ -172,7 +172,7 @@ std::vector<std::string> split(const std::string& s, char delimiter) {
 static int
 illixr_rt_launch(struct illixr_hmd *dh, const char *path, const char *comp, void* glctx)
 {
-	dh->runtime_lib = new ILLIXR::dynamic_lib{ILLIXR::dynamic_lib::create(path)};
+	dh->runtime_lib = new ILLIXR::dynamic_lib{ILLIXR::dynamic_lib::create((std::string) path)};
```

Comment:
Nice catch! I think this works, but let's use the `const char*` constructor from [`std::string`](https://en.cppreference.com/w/cpp/string/basic_string/basic_string), which is a bit safer than a C-style cast. We can use C++'s new [Uniform Initialization](https://arne-mertz.de/2015/07/new-c-features-uniform-initialization-and-initializer_list/) like suggested by [@mhuzai](https://github.com/ILLIXR/ILLIXR/issues/257#issuecomment-812662889).

Reply:
Good point. I'll get this updated.